src/runtime/schema.ts

```diff
@@ -70,14 +70,18 @@ export class Schema {
       } else if (field.kind === 'schema-collection') {
         return {kind: 'schema-collection', schema: updateField(field.schema)};
       } else {
-        return field;
+        const fieldLit = {...field};
```

Comment:
Does "Lit" stand for Literal, wasn't obvious initially? Maybe worth expanding or s/fieldLit/literal/ ?

Reply:
Thanks